apps/SageAccountWeb/src/hooks/useSkipMfaPrompt.ts

```diff
@@ -0,0 +1,44 @@
+import { SKIPPED_2FA_REQUIREMENT_TIMESTAMP_LOCALSTORAGE_KEY } from '@/utils/StorageKeys'
+import { useLocalStorageValue } from '@react-hookz/web'
+import dayjs from 'dayjs'
+import { useCallback, useMemo } from 'react'
+
+function isTimestampLessThan24HoursOld(timestamp: string | null): boolean {
+  try {
+    const twelveHoursAgo = dayjs().subtract(24, 'hours')
+    return timestamp !== null && dayjs(timestamp).isAfter(twelveHoursAgo)
+  } catch (error) {
+    console.error('Error thrown computing timestamp', timestamp, error)
+    return false
+  }
+}
+
+/**
+ * Hook to determine if the user has skipped the 2FA prompt recently, and provides a callback to invoke when the user
+ * skips MFA.
+ *
+ * This is temporary code that can be deleted once 2FA is required for all users.
+ * @param accountId
+ */
+export function useSkipMfaPrompt() {
+  const { value: lastSkippedTimestamp, set: setTimestamp } =
+    useLocalStorageValue<string>(
+      SKIPPED_2FA_REQUIREMENT_TIMESTAMP_LOCALSTORAGE_KEY,
+    )
```

Comment:
I originally wanted to include the account ID in the localstorage key, so you would be prompted for each account that logs in to the same machine. I ended up running into issues, I think race conditions w/ how `useLocalStorageValue` worked, so I ended up not doing that.